packages/editor/src/components/provider/index.js

```diff
@@ -164,6 +166,8 @@ class EditorProvider extends Component {
 				value={ blocks }
 				onInput={ resetEditorBlocksWithoutUndoLevel }
 				onChange={ resetEditorBlocks }
+				selectionStart={ selectionStart }
```

Comment:
In exposing this as a public interface:

- Where is the shape documented?
- Is it possible to consolidate this to a single prop, `selection` ?

Reply:
Right, it should be documented as part of this component as well. It's currently documented for a selector I believe.

I would not consolidate it as a single prop, to avoid nesting objects. In my opinion `start={ {} } end={ {} }` is better than `wrapped={ { start: {}, end: {} } }`. What do you think?

Reply:
`<textarea>` also has `selectionStart` and `selectionEnd`.